src/components/auth/Register2ndCard.vue

```diff
@@ -16,7 +16,11 @@
       label="last name"
     />
     <label class="register-2nd-card__datepicker-label">Date of Birth</label>
-    <button class="register-2nd-card__datepicker-button" @click="openPicker">
+    <button
+      type="button"
+      class="register-2nd-card__datepicker-button"
+      @click="openPicker"
+    >
       <slot class="register-2nd-card__datepicker-slot">
         {{ formData.dateOfBirth ? getDate(formData.dateOfBirth) : "" }}</slot
```

Comment:
What do you think about moving this line to the `computed` and then using it here ? :)